CMakeLists.txt

```diff
@@ -297,6 +297,25 @@ if(MINGW)
 	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mstackrealign -fno-strict-aliasing")
 endif()
 
+# support building on Raspberry Pi
+if(CMAKE_COMPILER_IS_GNUCXX AND CMAKE_SYSTEM_PROCESSOR STREQUAL "armv7l")
+  message(STATUS "Adding compiler flags for armv7l (RPi 2&3, BBB)")
+  foreach(flag CMAKE_C_FLAGS CMAKE_CXX_FLAGS)
+    set(${flag} "${${flag}} -mtune=cortex-a7")
+    set(${flag} "${${flag}} -mfloat-abi=hard")
+    set(${flag} "${${flag}} -mfpu=neon")
+    set(${flag} "${${flag}} -funsafe-math-optimizations")
```

Comment:
Just following @redFrik, he probably wanted to tweak the very slow RP1. Thanks for your input, both of you, will correct!